### PR TITLE
refactor(createTree): different dataType formats

### LINKED ISSUE #17: generalize convert tree
allow the use of `##DATA_TYPE =`, `##DATA TYPE=` or `##data type =`

### PR DESCRIPTION
**Pull request by maasencioh**
closes #17

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -888,8 +888,9 @@ function createTree(jcamp) {
             }
         } else if (current && current.jcamp) {
             current.jcamp += line + '\n';
-            if (line.substring(0, 10) === '##DATATYPE') {
-                current.dataType = line.substring(11).trim();
+            var dataType = line.match(/##data.*type\s*=\s*(.+)\s*/i);
```
Comment (lpatiny) [inline src/index.js]: It has to start with ## so I would put ^##

Please do it the way based on the specification

var match = line.match(/^##(.*?)=/);
if (match) {
  var dataLabel=match[1].replace(/[ _-]/g).toUpperCase();
  if (dataLabel==='DATATYPE') {

}
}

Comment (maasencioh) [reply src/index.js]: done

**Comment by targos** [comment]
is this ready?

**Comment by maasencioh** [comment]
@targos yes it is

**Comment by targos** [comment]
ping @lpatiny

**Pull request merged**